documentation/release-notes/topics/version_3.rst

```diff
@@ -1,6 +1,37 @@
 #########
 Version 3
 #########
+
+*****************************
+Hyperview 3.14 (June 7, 2023)
+*****************************
+This section covers significant changes and bug fixes in Hyperview 3.14.x since version 3.13.x.
+
+.. note:: This release has changes and improvements to the Data Collector software. It is strongly recommended to update your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+
+=======================================
+Enhanced Licensed Feature: Connectivity
+=======================================
+- RJ11 has been added as a connector type
+- Ports will be created automatically if they do not exist when importing connections
```

Comment:
It is more a "can" than a "will" since it requires the user to flip the setting to on.

Reply:
notes and addressed, thank you!